controllers/argocd/configmap.go

```diff
@@ -144,15 +144,46 @@ func getRBACScopes(cr *argoprojv1a1.ArgoCD) string {
 	return scopes
 }
 
-// getResourceCustomizations will return the resource customizations for the given ArgoCD.
+type RCdeprecationEventEmissionStatus struct {
+	ResourceCustomizationsDeprecationWarningEmitted bool
+}
+
+var RCdeprecationEventEmissionTracker = make(map[string]RCdeprecationEventEmissionStatus)
+
+// GetResourceCustomizations loads Resource Customizations from argocd-cm ConfigMap. Please note that ResourceCustomizations are being deprecated in favor of ResourceCustomizationsNew.
 func getResourceCustomizations(cr *argoprojv1a1.ArgoCD) string {
 	rc := common.ArgoCDDefaultResourceCustomizations
 	if cr.Spec.ResourceCustomizations != "" {
 		rc = cr.Spec.ResourceCustomizations
 	}
+	log.Info("You are using an old format for ResourceCustomizations, please use the new format.")
```

Comment:
This should ideally be a warning but I understand using the current logging implementation its not possible.

So, can we at least do this ?
	log.Info("[WARNING] You are using an old format for ResourceCustomizations, please use the new format.")
	
	and this in future with the zap logger implementation can be changed to `log.warn`.